truss-chains/truss_chains/remote_chainlet/utils.py

```diff
@@ -623,13 +638,19 @@ async def send_json(self, data: Any) -> None:
         await self._websocket.send_json(data)
 
     async def iter_text(self) -> AsyncIterator[str]:
-        while True:
-            yield await self.receive_text()
```

Comment:
Can we remove the `receive_text`, `receive_bytes`, and `receive_json` methods above?